common/src/main/java/org/jboss/hal/testsuite/cli/StandaloneManager.java

```diff
@@ -36,10 +39,21 @@
 
     private static Logger log = LoggerFactory.getLogger(StandaloneManager.class);
 
+    private Dispatcher dispatcher = new Dispatcher();
+
     public StandaloneManager(CliClient cliClient) {
         super(cliClient);
     }
 
+    public StandaloneManager(CliClient client, Dispatcher dispatcher) {
```

Comment:
Combining usage of CliClient and Dispatcher is dangerous, don't do that if possible. Also ServerManager classes are used from CliClient.